src/scalap/scala/tools/scalap/scalax/util/StringUtil.scala

```diff
@@ -10,7 +10,7 @@ import java.beans.Introspector
 
 object StringUtil {
 
-  def trimStart(s: String, prefix: String) = if (s != null && s.startsWith(prefix)) s.substring(prefix.length) else s
+  def trimStart(s: String, prefix: String) = if (s != null) s.stripPrefix(prefix) else s
```

Comment:
Could you please test whether we need that null check?
If not, you could probably replace `trimStart` with `stripPrefix` everywhere!